app/src/main/java/com/android/shelfLife/model/foodItem/FoodItemRepositoryFirestore.kt

```diff
@@ -0,0 +1,154 @@
+package com.android.shelfLife.model.foodItem
+
+import android.util.Log
+import com.google.firebase.Timestamp
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.DocumentSnapshot
+import com.google.firebase.firestore.FirebaseFirestore
+
+class FoodItemRepositoryFirestore(private val db: FirebaseFirestore) : FoodItemRepository {
+
+    private val collectionPath = "foodItems"
```

Comment:
It is a good practice to declare constant values in a companion object. This ensures that only one instance of the constant is created, reducing memory usage and avoiding unnecessary resource consumption.

```
companion object {
    private const val COLLECTION_PATH = "foodItems"
}   
```